storage/bucket.go

```diff
@@ -716,10 +721,12 @@ type LifecycleCondition struct {
 
 	// DaysSinceCustomTime is the days elapsed since the CustomTime date of the
 	// object. This condition can only be satisfied if CustomTime has been set.
+	// Note: Using `0` as the value will not be ignored by the library and not sent to the API.
```

Comment:
"will be ignored", not "will not be ignored". Same in L729 below.

Also, add this note to NumNewerVersions as well.